CHANGELOG.md

```diff
@@ -50,10 +50,10 @@ CHANGELOG
  - Inject `Pim\Bundle\DataGridBundle\Extension\MassAction\MassActionDispatcher` in `Pim\Bundle\CustomEntityBundle\Action\MassEditAction` instead of `Pim\Bundle\CustomEntityBundle\MassAction\DataGridQueryGenerator`
  - Inject `Pim\Bundle\DataGridBundle\Extension\MassAction\MassActionDispatcher`, `Pim\Bundle\ImportExportBundle\Entity\Repository\JobInstanceRepository`, `Akeneo\Bundle\BatchBundle\Launcher\JobLauncherInterface` and `Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface`
  in `Pim\Bundle\CustomEntityBundle\Action\QuickExportAction` of `Symfony\Bridge\Doctrine\RegistryInterface`, `Pim\Bundle\CustomEntityBundle\MassAction\DataGridQueryGenerator` and `Symfony\Component\Serializer\Serializer`
- - Repositories don't extend `Pim\Bundle\CatalogBundle\Doctrine\ReferableEntityRepository` neither implement `Pim\Bundle\UIBundle\Entity\Repository\OptionRepositoryInterface` anymore.
- - Methods `getOption`, `getOptionId`, `getOptionLabel` and `getOptions` has been removed from repositories
- - `Pim\Bundle\CustomEntityBundle\Entity\Repository\TranslatableCustomEntityRepository` don't implement Pim\Bundle\CustomEntityBundle\Entity\Repository\DatagridAwareRepositoryInterface` neither `Pim\Bundle\CustomEntityBundle\Entity\Repository\LocaleAwareRepositoryInterface`
- - Methods `createDatagridQueryBuilder` and `setLocale` and instance variable `locale` has been removed from `Pim\Bundle\CustomEntityBundle\Entity\Repository\TranslatableCustomEntityRepository`
+ - Repositories don't extend `Pim\Bundle\CatalogBundle\Doctrine\ReferableEntityRepository` nor implement `Pim\Bundle\UIBundle\Entity\Repository\OptionRepositoryInterface` anymore.
+ - Methods `getOption`, `getOptionId`, `getOptionLabel` and `getOptions` have been removed from repositories
+ - `Pim\Bundle\CustomEntityBundle\Entity\Repository\TranslatableCustomEntityRepository` doesn't implement `Pim\Bundle\CustomEntityBundle\Entity\Repository\DatagridAwareRepositoryInterface` nor `Pim\Bundle\CustomEntityBundle\Entity\Repository\LocaleAwareRepositoryInterface`
```

Comment:
or as well: "nor" is used:
 - with "neither" in a sentence like this `Pim\Bundle\CustomEntityBundle\Entity\Repository\TranslatableCustomEntityRepository` implements neither `Pim\Bundle\CustomEntityBundle\Entity\Repository\DatagridAwareRepositoryInterface` nor `Pim\Bundle\CustomEntityBundle\Entity\Repository\LocaleAwareRepositoryInterface`

- in a situation like this : "I don't like coffee, nor do I like tea". (And another person would respond "Neither do I").